support/k8s/k8s-workload-registrar/mode-crd/controllers/pod_controller.go

```diff
@@ -184,9 +204,44 @@ func (r *PodReconciler) podSpiffeID(pod *corev1.Pod) string {
 
 	// the controller has not been configured with a pod label or a pod annotation.
 	// create an entry based on the service account.
-	return makeID(r.c.TrustDomain, "ns/%s/sa/%s", pod.Namespace, pod.Spec.ServiceAccountName)
+	//return makeID(r.c.TrustDomain, "v1/provider/eu-de/%s/%s/%s", pod.Namespace, pod.Spec.ServiceAccountName, pod.Spec.Containers[0].Name)
+
+	// TODO setup some new variable to trigger the Identity Schema processing
+	newId := r.is.getSVID(pod)
+
+	return makeID(r.c.TrustDomain, newId)
+	// return makeID(r.c.TrustDomain, "ns/%s/sa/%s", pod.Namespace, pod.Spec.ServiceAccountName)
 }
 
 func (r *PodReconciler) podParentID(nodeName string) string {
 	return makeID(r.c.TrustDomain, "k8s-workload-registrar/%s/node/%s", r.c.Cluster, nodeName)
 }
+
+// podSpiffeID returns the desired spiffe ID for the pod, or nil if it should be ignored
+func (r *PodReconciler) getSelectors(pod *corev1.Pod) spiffeidv1beta1.Selector {
```

Comment:
this function seems a bit a out of place, config shouldnt be touched at this point probably. This function should probably check if `is` is not nil, use `is.getSelector` OR otherwise provide the default selectors from the workload registrar